frontends/web/src/routes/account/send/components/dialogs/scan-qr-dialog.tsx

```diff
@@ -0,0 +1,39 @@
+import { useTranslation } from 'react-i18next';
+import { Dialog } from '../../../../../components/dialog/dialog';
+import { Button } from '../../../../../components/forms';
+import { Spinner } from '../../../../../components/spinner/Spinner';
+import style from '../../send.module.css';
+
+type TProps = {
+    activeScanQR: boolean;
+    toggleScanQR: () => void;
+    videoLoading: boolean;
+    onLoadedVideo: () => void;
+}
+
+const DialogScanQR = ({ activeScanQR, toggleScanQR, videoLoading, onLoadedVideo }: TProps) => {
+  const { t } = useTranslation();
+  return (
+    <Dialog
+      open={activeScanQR}
+      title={t('send.scanQR')}
+      onClose={toggleScanQR}>
+      {videoLoading && <Spinner guideExists />}
```

Comment:
since you are here, I did a quick test to keep the spinner loading forever. 

Maybe out of scope for this PR, but the fullscreen spinner loading looks like the whole app quickly flickers. Also the guide has no entries or just doesn't work.

Feel free to keep as is or add a 2nd commit to refactor this.